README.md

```diff
@@ -2,6 +2,9 @@
 
 Fable bindings for virtual-dom
 
+## Website
+[fable.io/fable-arch](http://fable.io/fable-arch/#/)
```

Comment:
Can you update your PR to reflect our decision ?
Also please replace fable-import-virtualdom (old name) with Fable-arch :)